python/cm/stack.py

```diff
@@ -323,6 +324,10 @@ def save_import(proto, conf):
         if 'versions' in conf['import'][key]:
             check_versions(proto, conf['import'][key], f'import "{key}"')
             set_version(si, conf['import'][key])
+            if si.min_version and si.max_version:
+                if rpm.compare_versions(str(si.min_version), str(si.max_version)) > 0:
```

Comment:
Why `str()` function ?

Reply:
We need to pass str in function. str() helps with version looking like this (1.2.3.4), which can be type ScalarFloat.